Parsers/ASimProcess/ARM/ProcessEventGeneric/README.md

```diff
@@ -0,0 +1,15 @@
+# Source Agnostic ASIM ProcessEvent Normalization Parser
+
+This template deploys the Source Agnostic ASIM ProcessEvent schema parser. The parser is a part of the Azure Sentinel Information Model.
```

Comment:
This is the same as the Empty parser information, shouldn't we make it clear this is for Generic?